pkgs/applications/science/electronics/kicad/base.nix

```diff
@@ -25,13 +25,13 @@
 , stable ? true
 , baseName ? "kicad"
 , versions ? { }
-, oceSupport ? false
+, withOCE ? false
```

Comment:
Slight nitpick: The default values are already inherited from `default.nix` so it shouldn't be needed to specify them here as well, I think.

Reply:
It would be preferred to put out the defaults in `default.nix` and not list them here as well, as again - `default.nix` is the 1st thing a 3rd party reader reads.

Reply:
I removed all defaults in addition to the `with<feature>` ones since they're all provided in default.nix. Let me know if that's too far and I'll add those back in.

Reply:
No that's perfect.